apps/app/src/test/scala/org/lfdecentralizedtrust/splice/integration/tests/FeaturedAppActivityMarkerIntegrationTest.scala

```diff
@@ -58,92 +69,79 @@ class FeaturedAppActivityMarkerIntegrationTest
-                Seq(
-                  new featuredapprightv1.AppRewardBeneficiary(
-                    alice.toProtoPrimitive,
-                    BigDecimal(0.2).bigDecimal,
-                  ),
-                  new featuredapprightv1.AppRewardBeneficiary(
-                    charlie.toProtoPrimitive,
-                    BigDecimal(0.8).bigDecimal,
-                  ),
-                ).asJava
-              )
-              .commands
-              .asScala
-              .toSeq,
-          )
-
-        bobValidatorBackend.participantClientWithAdminToken.ledger_api_extensions.commands
-          .submitJava(
-            Seq(bob),
-            commands = bobFeaturedAppRightCid
-              .exerciseFeaturedAppRight_CreateActivityMarker(
-                Seq(
-                  new featuredapprightv1.AppRewardBeneficiary(
-                    bob.toProtoPrimitive,
-                    BigDecimal(1.0).bigDecimal,
-                  )
-                ).asJava
-              )
-              .commands
-              .asScala
-              .toSeq,
-          )
+        for (i <- 1 to markerMultiplier) {
+          aliceValidatorBackend.participantClientWithAdminToken.ledger_api_extensions.commands
+            .submitJava(
+              Seq(alice),
+              commands = aliceFeaturedAppRightCid
+                .exerciseFeaturedAppRight_CreateActivityMarker(
+                  Seq(
+                    new featuredapprightv1.AppRewardBeneficiary(
+                      alice.toProtoPrimitive,
+                      BigDecimal(0.2).bigDecimal,
+                    ),
+                    new featuredapprightv1.AppRewardBeneficiary(
+                      charlie.toProtoPrimitive,
+                      BigDecimal(0.8).bigDecimal,
+                    ),
+                  ).asJava
+                )
+                .commands
+                .asScala
+                .toSeq,
+            )
+          bobValidatorBackend.participantClientWithAdminToken.ledger_api_extensions.commands
+            .submitJava(
+              Seq(bob),
+              commands = bobFeaturedAppRightCid
+                .exerciseFeaturedAppRight_CreateActivityMarker(
+                  Seq(
+                    new featuredapprightv1.AppRewardBeneficiary(
+                      bob.toProtoPrimitive,
+                      BigDecimal(1.0).bigDecimal,
+                    )
+                  ).asJava
+                )
+                .commands
+                .asScala
+                .toSeq,
+            )
+        }
+        // unpause all activity marker triggers here, so they can start to get to work
+        env.svs.local.foreach(
+          _.dsoDelegateBasedAutomation.trigger[FeaturedAppActivityMarkerTrigger].resume()
+        )
       },
     )(
       "Activity markers are converted to reward coupons",
       _ => {
-        aliceValidatorBackend.participantClientWithAdminToken.ledger_api_extensions.acs.filterJava(
```

Comment:
removed the history checks as that is only dependent on proper handling of `AppRewardCoupon`s, which we check in other tests.